kolibri/plugins/learn/assets/src/views/ContentPage.vue

```diff
@@ -327,6 +357,12 @@
 
 <style lang="scss" scoped>
 
+  .folder-svg {
+    margin-right: 8px;
+    margin-bottom: -2px;
+    vertical-align: bottom;
+  }
```

Comment:
let's use the `KLabeledIcon` component here to avoid adding special-case layout styling.

If desired, we could can add a new prop to handle tooltips which would be valuable in other places also